tests/e2e/tests/pilot/authn_policy_test.go

```diff
@@ -27,25 +27,15 @@ func TestMTlsWithAuthNPolicy(t *testing.T) {
 		// The whole authn test suites should be rewritten after PR #TBD for better consistency.
 		t.Skipf("Skipping %s: authn=true", t.Name())
 	}
-	// This policy will enable mTLS globally (mesh policy)
-	globalCfg := &deployableConfig{
```

Comment:
It's better to use option from https://github.com/istio/istio/pull/7778 to 'modify' the global policy rather than bend the test. We want to keep at least one test for the global mTLS enablement.